applications/launchpad/gui-react/__tests__/mocks/states/miningStateTemplates.ts

```diff
@@ -2,45 +2,32 @@ import { MiningState } from '../../../src/store/mining/types'
 
 export const initialMining: MiningState = {
   tari: {
-    sessions: [],
+    session: undefined,
   },
   merged: {
-    sessions: [],
+    session: undefined,
   },
 }
 
 export const miningWithSessions: MiningState = {
   tari: {
-    sessions: [
-      {
-        total: {
-          xtr: '1000',
-        },
+    session: {
+      startedAt: Number(Date.now()).toString(),
+      total: {
+        xtr: '1000',
       },
-      {
-        total: {
-          xtr: '2000',
-        },
-      },
-    ],
+    },
   },
   merged: {
     threads: 1,
     urls: ['firstAddress'],
     address: 'address',
-    sessions: [
-      {
-        total: {
-          xtr: '1000',
-          xmr: '1001',
-        },
-      },
-      {
-        total: {
-          xtr: '2000',
-          xmr: '2001',
-        },
+    session: {
+      startedAt: Number(Date.now()).toString(),
+      total: {
+        xtr: '1000',
+        xmr: '1001',
       },
-    ],
+    },
   },
```

Comment:
Changing the session type from array to single object. Probably we won't store the history of sessions here so array is not handful.